mirr254/gluu-server-helm
# Create config-init microservice helm chart

### What does this PR do?
- Creates helm chart for initial configuration required to run the Gluu server

### Description of work done
- Create a local dependancy chart for config initialization
- Create tiller service account in the `kube-system` namespace
- Create ClusterRoleBinding for service account and bind it with cluster-admin user. This will give helm access across all namespaces.
- Create a `README.md` for a brief explanation of the thought process

### How should this be manually tested
- Make sure `helm` and 'kubectl` are installed on your cluster
- clone this repo `cd gluu-server-helm` and `checkout` to `ft-create-config-roles` branch
- run `helm init` followed by `helm install gluu-server-helm`

### Screenshots
<img width="1049" alt="Screenshot 2019-07-03 at 12 16 02" src="https://user-images.githubusercontent.com/13331207/60579733-619a5f80-9d8c-11e9-9a26-fb597e74b3e2.png">
